### PR TITLE
Update go-cam-shapes.shex - remove CellularComponent when also have AnatomicalEntity

### PR DESCRIPTION
**Pull request by vanaukenk**
Removing 'OR \<CellularComponent> from occurs_in relation on BP and MF shape.

We only need to use 'AnatomicalEntity' here now that GO CC is defined as a 'cellular anatomical entity'

### CODE REVIEW

**Comment by vanaukenk** [comment]
If I'm reading the Details report correctly, it looks like this pull request failed because the GO CC 'host cell cytoplasm' is not defined as a 'cellular anatomical entity'.
'host cell cytoplasm' is used in a multi-organism test model.
@ukemi @pgaudet @balhoff

**Comment by goodb** [comment]
@vanaukenk are you used to the travis reporting system here?  let me know if not.  It looks like there is a failure on an occurs_in statement in test-multi-org-1.ttl:

test-multi-org-1.ttl conformant:false
4847test-multi-org-1.ttl should have validated but did not report type id = SHEX_CORE_SCHEMA
4848rulefile = https://github.com/geneontology/go-shapes/blob/master/shapes/go-cam-shapes.shex
4849tracker = https://github.com/geneontology/go-shapes/issues
48501 noncomformant nodes detected:
4851node: http://model.geneontology.org/5d29221b00001265/5d29221b00001267 was expected to match shape: http://purl.obolibrary.org/obo/go/shapes/MolecularFunction but did not fit the following constraints:
4852	the objects of assertions made with http://purl.obolibrary.org/obo/BFO_0000066 should be nodes that fit the one of these shapes: 
4853		[http://purl.obolibrary.org/obo/go/shapes/AnatomicalEntity]
4854		but, sadly, the object http://model.geneontology.org/5d29221b00001265/5d29221b00001281 of one such assertion emanating from the failing node here did not.

**Comment by vanaukenk** [comment]
@goodb - yes, that's what I was looking at, but now I'm not so sure of my interpretation.

**Comment by goodb** [comment]
Looks like it doesn't think 'host cell cytoplasm' GO_0030430  is an anatomical entity.  I will check the validator to make sure it has the most recent ontology in mind.

**Comment by vanaukenk** [comment]
Yes, I think my initial interpretation was wrong, but then I'm not sure why the pull request failed.

**Comment by vanaukenk** [comment]
Ah, okay.  I did just check in Protege and it looks there like 'host cell cytoplasm' is considered an anatomical entity.

**Comment by goodb** [comment]
@vanaukenk @kltm the validator is using rdf.geneontology.org to get the subclass closure for the GO.  So far it seems that @balhoff 's change hasn't percolated there yet.  

select ?c ?label where {
  <http://purl.obolibrary.org/obo/GO_0030430> rdfs:subClassOf* ?c .
  ?c rdfs:label ?label 
     }


<http://purl.obolibrary.org/obo/BFO_0000002> | continuant
-- | --
<http://purl.obolibrary.org/obo/BFO_0000004> | independent continuant
<http://purl.obolibrary.org/obo/BFO_0000040> | material entity
<http://purl.obolibrary.org/obo/BFO_0000040> | material entity
<http://purl.obolibrary.org/obo/GO_0005575> | cellular_component
<http://purl.obolibrary.org/obo/GO_0018995> | host cellular component
<http://purl.obolibrary.org/obo/GO_0030430> | host cell cytoplasm
<http://purl.obolibrary.org/obo/GO_0033643> | host cell part
<http://purl.obolibrary.org/obo/GO_0033646> | host intracellular part

**Comment by vanaukenk** [comment]
Thanks @goodb !

**Comment by goodb** [comment]
I'd like to change it so that it downloads the ontology fresh for the validation, but this isn't going to work unless we get more memory allocated to this travis instance (or we change how neo is handled).

**Comment by goodb** [comment]
@vanaukenk should the constraint on MF be 
occurs_in: ( `@<AnatomicalEntity>` ) {0,1};  
as it is in this branch or 
occurs_in: ( `@<AnatomicalEntity>` OR `@<CellularComponent>` )
as in master and vanauken-patch-3  ?

We have much of CellularComponent subclassed under AnatomicalEntity now, but not complexes.  If we run with occurs_in: ( `@<AnatomicalEntity>)` then the should_pass file Test004-occurs_in_GO_Complex.ttl fails validation as a function can't occur in a complex.  

So in a nutshell here.  Should it be possible for a function (or process) to occur in a complex?  If yes, should the ontology have complex as a subclass of anatomical entity or should we add an OR statement to the shex here like:
occurs_in: ( `@<AnatomicalEntity>` OR `@<ProteinComplex>` )

ping @balhoff
